src/analysis/normed_space/star/exponential.lean

```diff
@@ -26,21 +26,15 @@ variables {A : Type*}
 
 open complex
 
-lemma is_self_adjoint.exp_i_smul_unitary {a : A} (ha : is_self_adjoint a) :
-  exp ℂ (I • a) ∈ unitary A :=
-begin
-  rw [unitary.mem_iff, star_exp],
-  simp only [star_smul, is_R_or_C.star_def, self_adjoint.mem_iff.mp ha, conj_I, neg_smul],
-  rw ←@exp_add_of_commute ℂ A _ _ _ _ _ _ ((commute.refl (I • a)).neg_left),
-  rw ←@exp_add_of_commute ℂ A _ _ _ _ _ _ ((commute.refl (I • a)).neg_right),
-  simpa only [add_right_neg, add_left_neg, and_self] using (exp_zero : exp ℂ (0 : A) = 1),
-end
+lemma is_self_adjoint.I_smul_mem_skew_adjoint {a : A} (ha : is_self_adjoint a) :
+  (I • a) ∈ skew_adjoint A :=
+by rw [skew_adjoint.mem_iff, star_smul, is_R_or_C.star_def, conj_I, ha.star_eq, neg_smul]
```

Comment:
This belongs much lower in the hierarchy, preferably in `data.is_R_or_C.basic` unless it can go earlier. I don't think the porting tide has reached there yet so we shouldn't need a forward port.

Also, there should be a few lemmas of this flavor (e.g., this could be an iff, but the version you have is nice for dot notation). Could you provide the iff versions and then just `alias` the others for dot notation? I'm thinking `I_smul_mem_self_adjoint_iff` and `I_smul_mem_skew_adjoint_iff` with your lemma and `is_skew_adjoint.I_smul_mem_self_adjoint` as aliases.

Reply:
Note that there is no `is_skew_adjoint`

Reply:
Oh right, so three lemmas then. The one you have